Makefile

```diff
@@ -20,30 +24,58 @@ bundle-docs: bundle-docs-clean
 	doc2dash --name pyteal --index-page index.html --online-redirect-url https://pyteal.readthedocs.io/en/ _build/html && \
 	tar -czvf pyteal.docset.tar.gz pyteal.docset
 
+# ---- Code Quality ---- #
+
 generate-init:
```

Comment:
@tzaffi Minor and optional:  Does `generate-init` fall under the _code quality_ or _setup_ umbrella?  I thought it's the latter because `build-and-test` fails otherwise.

I don't feel strongly.  If you prefer as is, feel welcomed to resolve.

Reply:
👍